browsers/opera_android.json

```
@@ -51,16 +52,19 @@
           "engine_version": "26"
         },
         "15": {
+          "release_date": "2013-07-02",
           "status": "retired",
           "engine": "Blink",
           "engine_version": "28"
         },
         "16": {
+          "release_date": "2013-08-26",
```

This appears to be a little later, September 18 2013

https://web.archive.org/web/20130921055319/http://my.opera.com/chooseopera/blog/2013/09/18/opera-16-for-android-is-released-2

```suggestion
          "release_date": "2013-09-18",
```